tools/scripts/build_ign.sh

```diff
@@ -21,19 +22,22 @@ sudo apt -y install \
 mkdir build
 cd build
 cmake ../ -DBUILD_TESTING=false
-sudo make install
+make doc
```

Comment:
I believe the `make install` option was used in order to produce documentation that includes correct cross-links.

Reply:
Yeah that's what I though. But that should work well using debs too, right? The only caveat being that some cross-links may be out of date until we release debs for the dependencies.

Alternatively, I considered adding a target like `doc-install` which installs only the doc stuff, so we don't need to build all the source code.

Reply:
That's right. Using debians will only have docs based on the latest release. I'm fine with that.